js/src/cli.js

```diff
@@ -602,10 +607,31 @@ function checkFiles(parsed) {
     parsed.replace = true;
   }
 
+  // support glob
+  var hasGlob = parsed.glob !== undefined;
+
+  if (hasGlob && parsed.files.length) {
+    // Filter the files with glob (also required for a git precommit hook)
+    miniMatch = new Minimatch(parsed.glob);
```

Comment:
Rather than minimatching, what do you think of what I said above?  Expanding the files array with globs, check if the result is empty, and finally removing duplicates.

Reply:
For all comments:
**Current situation:**
I'm using Prettier to format TypeScript code, but Prettier doesn't support HTML, so I want to use beautify-js for this, but for the HTML files only!
The --type html doesn't filter HTML files only? (or did I do something wrong?)

**Sugestion:**
After this PR is finished I want to make a pre-commit option (without the need for other packages like glob-run as sugested [here](https://github.com/beautify-web/js-beautify/issues/787#issuecomment-315628380).
The files from the pre-commit option will be added before this minimatch filter, so we are still be able to run a filter on these files. (but again, maybe I did something wrong with the --type option)

What do you think of this or how can we handle my scenario?

Reply:
Well, if globbing works the way I'd expected, you'd call:
```
js-beautify --type html  *.html *.htm 
```

But I get the sense you had a different idea.   Could you give some examples?